src/main/java/uk/gov/hmcts/reform/sendletter/services/ReportsService.java

```diff
@@ -1,16 +1,72 @@
 package uk.gov.hmcts.reform.sendletter.services;
 
-import org.apache.commons.lang.NotImplementedException;
+import org.springframework.beans.factory.annotation.Value;
 import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.sendletter.entity.LettersCountSummaryRepository;
+import uk.gov.hmcts.reform.sendletter.entity.reports.ServiceLettersCountSummary;
 import uk.gov.hmcts.reform.sendletter.model.out.LettersCountSummary;
+import uk.gov.hmcts.reform.sendletter.services.ftp.ServiceFolderMapping;
 
 import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+import java.time.ZoneId;
+import java.time.ZonedDateTime;
+import java.time.format.DateTimeFormatter;
 import java.util.List;
 
+import static java.util.stream.Collectors.toList;
+import static uk.gov.hmcts.reform.sendletter.util.TimeZones.EUROPE_LONDON;
+
 @Service
 public class ReportsService {
 
+    private static final String TEST_SERVICE = "BULKPRINT";
```

Comment:
it's `send_letter_tests`

Reply:
I'm using the folder names in the report

Reply:
that's correct I think else you would have `divorce_frontend` for divorce

Reply:
The model suggests it's a service name @aliveni 
```
public class LettersCountSummary {

    public final String service;
    public final int uploaded;

    public LettersCountSummary(String service, int uploaded) {
        this.service = service;
        this.uploaded = uploaded;
    }
}
```
I assume business is interested in service names, not folders on FTP server.

Reply:
I'm leaving the model as it is for now as we will new config for the service names